hazelcast/src/test/java/com/hazelcast/test/annotation/LiveTest.java

```diff
@@ -0,0 +1,10 @@
+package com.hazelcast.test.annotation;
+
+
+/**
+ * Annotates tests which requires integration tests over a live api.
+ * <p/>
+ * Will be executed in nightly builds and for jclouds live tests
+ */
+public final class LiveTest {
```

Comment:
@Donnerbart Yes, you are right using jclouds own `ComputeService` has problems when provisioning multiple machines, I could not even start more than two machines on google-compute engine, the tests just hangs and gives timeout exception. But on AWS, I did not encounter reliability issues , but I am pretty sure it will create problems during running automated tests. Actually what I want is provide more live tests for different cloud vendors easily so that we can see problems due to API changes ahead of time with the help of live tests, so depending on AWS CLI tool might be problem, what I want is a generic provisioning tool for different cloud vendors which is jclouds. But it seems, this makes our code coverage built dependent of other APIs. Maybe we might use your approach, we also used heavy mocking for unit tests without any dependency to AWS and maybe we can put all live tests in separate repo and run them on different built once a week. We need to have same kind of verification so that in each release we are sure that our code is working over Live API.